packages/edit-site/src/components/sidebar-navigation-item/style.scss

```diff
@@ -0,0 +1,15 @@
+.edit-site-sidebar-navigation-item.components-item {
+	color: $gray-500;
```

Comment:
Just curious about how this PR is working so I checked it out, and the build failed for me because this variable is undefined. Changing it to `$gray-600` or something else [defined here](https://github.com/WordPress/gutenberg/blob/trunk/packages/base-styles/_colors.scss#L7-L14) fixed it.